.github/workflows/k8s-test-run.yml

```diff
@@ -0,0 +1,26 @@
+name: Weekly k8s test run
+on:
+  schedule:
+    # At 16:00:00 - UTC, every week on Monday, every month.
+    - cron: '0 16 * * MON'
```

Comment:
let's put it earlier in the day(9-10 UTC)  - if it happens in the evening the on-call guys get notifications and it's not good.